docker/cron/stats-crontab

```diff
@@ -57,9 +59,3 @@
 
 # user weekly daily_activity
 10 03 * * * listenbrainz_stats_cron /usr/local/bin/python /code/listenbrainz/manage.py spark request_user_stats --type=daily_activity --range=all_time
-
```

Comment:
This process is no longer running -- it keeps importing the same old data, so stop it for now.